embedded-uart-common/sample-implementations/linux/sensirion_uart_implementation.c

```diff
@@ -97,7 +97,7 @@ int16_t sensirion_uart_open() {
     return 0;
 }
 
-int16_t sensirion_uart_release() {
+int16_t sensirion_uart_close() {
```

Comment:
Could you add this to the changelog since people might want to know..

Reply:
It's only in the implementation wrong the header file is correct, but yes I will add it.